src/richie/apps/core/templates/richie/icons.html

```diff
@@ -181,8 +181,20 @@
+    </symbol>
+
```

Comment:
Here it is a compressed version of your icons and with a fix about fill (`#000` -> `currentColor`)
```suggestion
<symbol id="icon-languages" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
      <path fill-rule="nonzero" fill="currentColor" d="M12 22c-1.398 0-2.707-.262-3.926-.79a10.082 10.082 0 0 1-3.176-2.136 9.93 9.93 0 0 1-2.125-3.187C2.258 14.664 2 13.352 2 11.949c0-1.398.258-2.703.773-3.91a9.998 9.998 0 0 1 2.125-3.164 9.826 9.826 0 0 1 3.176-2.113C9.293 2.254 10.602 2 12 2c1.398 0 2.707.254 3.926.762a9.826 9.826 0 0 1 3.176 2.113 9.998 9.998 0 0 1 2.125 3.164c.515 1.207.773 2.512.773 3.91 0 1.403-.258 2.715-.773 3.938a9.93 9.93 0 0 1-2.125 3.187 10.082 10.082 0 0 1-3.176 2.137C14.707 21.738 13.398 22 12 22Zm0-1.45a8.38 8.38 0 0 0 1.46-2.062c.395-.777.716-1.695.966-2.761H9.602c.23 1 .543 1.898.937 2.699.39.8.879 1.508 1.461 2.125Zm-2.125-.3a11.863 11.863 0 0 1-1.074-2.05c-.301-.735-.551-1.56-.75-2.473H4.3c.633 1.183 1.367 2.109 2.199 2.785.832.675 1.957 1.254 3.375 1.738Zm4.273-.023a10.057 10.057 0 0 0 3.239-1.727 8.898 8.898 0 0 0 2.312-2.773h-3.722a18.318 18.318 0 0 1-.766 2.449c-.29.734-.645 1.414-1.063 2.05Zm-10.347-6h3.972a15.235 15.235 0 0 1-.085-1.215c-.008-.36-.012-.711-.012-1.063 0-.414.008-.785.023-1.113.016-.324.051-.688.102-1.086h-4a7.358 7.358 0 0 0-.239 1.074A8.525 8.525 0 0 0 3.5 11.95c0 .434.02.82.063 1.164.042.34.12.711.238 1.114Zm5.523 0H14.7c.067-.52.11-.938.125-1.266.016-.324.028-.66.028-1.012 0-.332-.012-.652-.028-.96A17.091 17.091 0 0 0 14.7 9.75H9.324c-.066.516-.11.93-.125 1.238a20.997 20.997 0 0 0 0 1.973c.016.328.059.746.125 1.266Zm6.875 0h4c.117-.403.196-.774.238-1.114a9.47 9.47 0 0 0 .063-1.164c0-.433-.02-.808-.063-1.125A7.358 7.358 0 0 0 20.2 9.75h-3.972c.046.582.082 1.027.097 1.336.016.309.028.598.028.863 0 .367-.016.715-.04 1.04a45.43 45.43 0 0 1-.113 1.238Zm-.25-5.977h3.75a8.126 8.126 0 0 0-2.262-2.875 8.623 8.623 0 0 0-3.312-1.625c.418.617.77 1.285 1.063 2 .292.715.546 1.55.761 2.5Zm-6.347 0h4.847a9.75 9.75 0 0 0-.926-2.563A10.284 10.284 0 0 0 12 3.5c-.535.45-.984 1.043-1.352 1.773-.363.735-.714 1.727-1.046 2.977Zm-5.301 0h3.773c.184-.898.418-1.703.7-2.414.285-.707.644-1.395 1.078-2.063-1.25.317-2.344.852-3.278 1.602-.933.75-1.691 1.707-2.273 2.875Zm0 0" />
    </symbol>

    <symbol id="icon-groups" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
      <path fill-rule="nonzero" fill="currentColor" d="M0 18v-1.324c0-.645.348-1.168 1.04-1.57.69-.403 1.597-.606 2.718-.606.203 0 .398.004.586.012.187.008.37.027.554.054-.132.29-.23.582-.296.88a4.238 4.238 0 0 0-.102.929V18Zm6 0v-1.625c0-.535.145-1.02.438-1.46.292-.442.703-.833 1.238-1.165.531-.332 1.172-.582 1.91-.75.742-.168 1.547-.25 2.414-.25.883 0 1.695.082 2.438.25.742.168 1.378.418 1.914.75.53.332.941.723 1.222 1.164.285.441.426.926.426 1.461V18Zm13.5 0v-1.625a4.78 4.78 0 0 0-.086-.938 3.93 3.93 0 0 0-.289-.87c.184-.028.367-.047.555-.055.187-.008.375-.012.57-.012 1.125 0 2.031.2 2.719.594.687.398 1.031.922 1.031 1.582V18Zm-12-1.5h9v-.148c0-.618-.422-1.122-1.262-1.516-.843-.39-1.922-.586-3.238-.586s-2.395.195-3.238.586c-.84.394-1.262.906-1.262 1.539Zm-3.762-2.75c-.476 0-.883-.172-1.226-.516A1.687 1.687 0 0 1 2 12c0-.484.172-.895.516-1.238.34-.34.754-.512 1.234-.512.484 0 .895.172 1.238.512.34.343.512.758.512 1.25 0 .476-.172.883-.512 1.226a1.71 1.71 0 0 1-1.25.512Zm16.5 0c-.476 0-.883-.172-1.226-.516A1.687 1.687 0 0 1 18.5 12c0-.484.172-.895.516-1.238.34-.34.754-.512 1.234-.512.484 0 .895.172 1.238.512.34.343.512.758.512 1.25 0 .476-.172.883-.512 1.226a1.71 1.71 0 0 1-1.25.512ZM12 12a2.898 2.898 0 0 1-2.125-.875A2.898 2.898 0 0 1 9 9c0-.852.293-1.563.875-2.137C10.457 6.29 11.168 6 12 6c.852 0 1.563.29 2.137.863C14.71 7.437 15 8.148 15 9c0 .832-.29 1.543-.863 2.125-.575.582-1.285.875-2.137.875Zm.008-1.5c.43 0 .785-.145 1.066-.434a1.47 1.47 0 0 0 .426-1.074c0-.43-.145-.785-.43-1.066a1.443 1.443 0 0 0-1.062-.426c-.422 0-.778.145-1.07.43-.293.285-.438.64-.438 1.062 0 .422.145.778.434 1.07.289.293.648.438 1.074.438Zm-.008 6ZM12 9Zm0 0" />
    </symbol>
```

Reply:
Wow, how you did that? Are you using some tool?

Thanks

Reply:
Yup, I use https://jakearchibald.github.io/svgomg/